acceptance/indy_acceptance/indy-tests/test_scripts/reporter.py

```diff
@@ -0,0 +1,519 @@
+"""
+Created on Nov 12, 2017
+
+@author: nghia.huynh
+
+Containing all functions and classes to make a HTML report.
+"""
+import os
+import json
+import socket
+import platform
+import glob
+import sys
+import subprocess
+import errno
+
+
+class FileNameGetter:
```

Comment:
In Python there is no necessity to use classes as containers of static members as it would be in Java.
In Python a module is a natural container for static data and functions.